tests/snappi_tests/multidut/pfc/files/multidut_helper.py

```diff
@@ -77,58 +87,128 @@ def run_pfc_test(api,
     stop_pfcwd(duthost2, tx_port['asic_value'])
     disable_packet_aging(duthost2)
 
-    """ Rate percent must be an integer """
-    test_flow_rate_percent = int(TEST_FLOW_AGGR_RATE_PERCENT / len(test_prio_list))
+    global DATA_FLOW_DURATION_SEC
+    global data_flow_delay_sec
+
+    dut_port = snappi_extra_params.multi_dut_params.multi_dut_ports[0]['peer_port']
+
+    port_id = 0
+    pytest_assert(port_id is not None,
```

Comment:
Here `port_id` will never be None since you're setting it to 0 in the previous line. What's the point of the `pytest_assert` here?

Reply:
this has been fixed